pkg/parse/run.go

```diff
@@ -419,21 +442,21 @@ func updateSyncStatus(ctx context.Context, p Parser) {
 
 		case <-updateTimer.C:
 			klog.V(3).Info("Updating sync status (periodic while syncing)")
-			if err := p.SetSyncStatus(ctx, p.options().updater.applier.Errors()); err != nil {
+			if err := setSyncStatus(ctx, p, state, p.SyncErrors()); err != nil {
 				klog.Warningf("failed to update sync status: %v", err)
 			}
-			remediatorErrs := p.RemediatorConflictErrors()
-			if len(remediatorErrs) > 0 {
-				UpdateConflictManagerStatus(ctx, remediatorErrs, p.K8sClient())
-			}
 
 			updateTimer.Reset(updatePeriod) // Schedule status update attempt
 		}
 	}
 }
 
-// UpdateConflictManagerStatus reports the conflict in the conflicting manager.
-func UpdateConflictManagerStatus(ctx context.Context, conflictErrs []status.ManagementConflictError, k8sClient client.Client) {
+// reportRootSyncConflicts reports conflicts to the RootSync that manages the
+// conflicting resources.
+func reportRootSyncConflicts(ctx context.Context, k8sClient client.Client, conflictErrs []status.ManagementConflictError) error {
```

Comment:
I don't know if this name change is critical. I can revert it if you like the old name better. But the old name made me think THIS RootSync was being updated and not some other RootSync. I used "report" instead of "update" because it seemed more like something you might do to a remote object. I also made it lowercase because now that the async status update is here, it's only used in this file, so it doesn't need to be public any more.

Reply:
sounds good